scrapbook-image-service/src/main/java/com/iu/scrapbook/document/Image.java

```diff
@@ -0,0 +1,24 @@
+package com.iu.scrapbook.document;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * @author jbhushan
+ */
+@Document
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Image {
+
+    @Id
+    private Long id;
```

Comment:
Isn't it better to use UUID instead of Long?

Reply:
Anything works as long as they all are unique. UUID is difficult to read so we generally avoids using it. But here we are not going to sho id to user so we can use UUID. I will change that.